setup.py

```diff
@@ -186,6 +186,8 @@ def compile_KLU():
         "scikit-fem>=0.2.0",
         "casadi>=3.5.0",
         "imageio>=2.9.0",
+        "julia>=0.5.6",
+        # Can be installed even if julia is not installed
```

Comment:
Does this line refer to the packages below or was there an additional julia package?

Reply:
This refers to the julia package, the pip package can be installed even if the julia programming language is not installed (so there is no need to make the pip package an optional dependency). Edited the comment